frontend/src/scenes/data-warehouse/settings/source/SourceConfiguration.tsx

```diff
@@ -26,24 +26,33 @@ interface UpdateSourceConnectionFormContainerProps extends SourceFormProps {
     id: string
 }
 
+const supportedEditingSourceTypes = ['MSSQL', 'MySQL', 'Postgres', 'Stripe']
```

Comment:
Nit: could this be typed with `ExternalDataSourceType[]` to ensure no future typos?

Reply:
Sure, we can type this.